src/components/Layout/Layout.scss

```diff
@@ -1,7 +1,3 @@
-.title {
-  color: tomato;
-}
-
 .layout {
-  margin: 50px;
+  margin: 0px;
```

Comment:
margin: 0;

(px are extra characters here that we need to ship to client to get the same work done)